pennylane/qchem/openfermion_pyscf.py

```diff
@@ -1237,3 +862,35 @@ def _pyscf_integrals(
         two_mo = two_mo[qml.math.ix_(active, active, active, active)]
 
     return core_constant, one_mo, two_mo
+
+
+def _openfermion_hamiltonian(
```

Comment:
Good to have at least a single-line docstring for private methods as well.

Reply:
Added a short one.